templates/react.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component, PropTypes } from 'react'
+
+export default class extends Component {
+  static propTypes = {
+    color: PropTypes.string,
+    hoverColor: PropTypes.string
+  };
+
+  constructor () {
+    super()
+    this.displayName = '__NAME__'
+    this.state = { isHovering: false }
+  }
+
+  render () {
+    const color = this.state.isHovering ? this.props.hoverColor : this.props.color
```

Comment:
If all the icons were set to `currentColor` you would easily be able to change their color from the parent component using the CSS `color` property. It also has the benefit that the icons will automatically inherit the surrounding text color (including it's hover/focus/active color). 😉